ietf/static/css/ietf.scss

```diff
@@ -236,9 +236,12 @@ th,
 
 // Helper to constrain the size of the main logo
 .ietflogo {
-    width: 75%;
+    width: 100%;
     max-width: 300px;
```

Comment:
would it make sense to set a min-width (of, perhaps, `100px`) here as well?

Reply:
Ok